docs/workflow.rst

```diff
@@ -22,22 +22,83 @@ template is the 'metal_production' subdirectory in beast/examples.
 
 In this location, at a minimum you will need the following files:
 
-  * datamodel.py
   * run_beast_production.py: a "production" version of run_beast.py
         - Provides commandline options for sub region files
+  * beast_production_wrapper.py: assembles the commands below into a script
+  * datamodel_template.py: a "production" version of datamodel.py that
+    will have name/filter fields automatically filled in by beast_production_wrapper
   * symbolic link to the beast directory in the beast repository
 
   .. code:: shell
 
      $ ln -s /location/beast/beast/ beast
 
-Datamodel.py
-============
+
+
+Datamodel_template.py
+===============
```

Comment:
This one of the lines that is giving a warning.  The '===' have to have the same number of characters as the label above it.